.github/workflows/circuit_test.yaml

```diff
@@ -0,0 +1,21 @@
+name: circuit test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: nargo test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Nargo
+        uses: noir-lang/noirup@v0.1.3
+        with:
+          toolchain: 0.22.0
```

Comment:
We can't use `0.21.0` because it was not released for linux https://github.com/noir-lang/noir/releases/tag/v0.21.0